Sources/Sentry/SentrySwiftAsyncIntegration.m

```diff
@@ -0,0 +1,17 @@
+#import "SentrySwiftAsyncIntegration.h"
+#import "SentryCrashStackCursor_SelfThread.h"
+
+@implementation SentrySwiftAsyncIntegration
+
+- (BOOL)installWithOptions:(nonnull SentryOptions *)options
+{
+    sentrycrashsc_setSwiftAsyncStitching(options.stitchSwiftAsync);
```

Comment:
`h`: This will enable the feature for everyone, if I'm not mistaken. We need to do what we do for every integration https://github.com/getsentry/sentry-cocoa/blob/4053ee90b75473db21ccfc2e1f361f22e2d0544c/Sources/Sentry/SentryScreenshotIntegration.m#L23-L25

Reply:
In the next line the code returns `stitchSWiftAsync` which means the integration will not be installed if the feature is disabled. The default is `NO`

Reply:
Isn't calling super essential for the list of integrations sent with every event?

Reply:
No. `installWithOptions` is just a check that returns true or false whether the integration should be installed.
SentrySDK is responsible for the magic

https://github.com/getsentry/sentry-cocoa/blob/2405ba598c9bae766901b68fd3cf1a9c5dca97af/Sources/Sentry/SentrySDK.m#L356-L361